dashboard/db/schema.rb

```diff
@@ -486,7 +486,7 @@
     t.index ["library_name", "library_version", "question_name"], name: "index_foorm_library_questions_on_multiple_fields", unique: true
   end
 
-  create_table "foorm_simple_survey_forms", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci", force: :cascade do |t|
+  create_table "foorm_simple_survey_forms", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci", force: :cascade do |t|
```

Comment:
There's no migration in this PR (right?) so we probably don't want to change schema.rb.  (There's a [long standing issue](https://codedotorg.atlassian.net/browse/FND-1398) that causes the utf8mb4 vs utf8 mismatch.)

Reply:
oh whoops, yeah this shouldn't be in the pr